views/js/controller/Delivery/monitoring.js

```diff
@@ -452,6 +448,78 @@ define([
                 }
 
                 /**
+                 * Create a warning message for execBulkAction()
+                 * @param {String} action
+                 * @param {String[]} selection
+                 * @param {Object[]} deniedResources
+                 */
+                function buildWarningMessage(action, selection, deniedResources) {
+                    var maxReasons = 2;
+                    var warningAction;
+                    var warningReason;
+                    var warningReasonOmission = __('...');
+
+                    deniedResources = deniedResources || [];
+                    selection = selection || [];
+
+                    warningAction = getWarningAction(action, deniedResources.length > 1 || selection.length > 1);
+                    warningReason = _(deniedResources || [{}])
+                        .slice(0, maxReasons)
+                        .map(function (deniedResource, index) {
+                            return getWarningReason(deniedResource.reason, deniedResource.id || (index + 1));
+                        })
+                        .join(' ');
+
+                    return __('%s %s%s', warningAction, warningReason, (deniedResources.length > maxReasons ? warningReasonOmission : ''));
```

Comment:
FYI, if you need a format string, you could also rely on the module `core/format`. This is is the same helper used by the translate helper to format parametric translations.
So here you'd just need to replace `__` by `format`, after having required the proper module of course.